#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_mythmusic_endian.ppc.dpatch by Magnus Ihse Bursie <magnus@ihse.net>
##
## All lines beginning with ## DP:' are a description of the patch.
## DP: Fix wrong path for data

@DPATCH@

--- mythplugins-0.20.orig.mythmusic/mythmusic/globalsettings.cpp	2006-09-13 09:54:08.000000000 +0200
+++ mythplugins-0.20/mythmusic/mythmusic/globalsettings.cpp	2006-09-13 09:55:15.000000000 +0200
@@ -19,7 +19,7 @@
 {
     HostLineEdit *gc = new HostLineEdit("MusicLocation");
     gc->setLabel(QObject::tr("Directory to hold music"));
-    gc->setValue("/mnt/store/music/");
+    gc->setValue("/var/lib/music/");
     gc->setHelpText(QObject::tr("This directory must exist, and the user "
                     "running MythMusic needs to have write permission "
                     "to the directory."));
